### ISSUE TITLE
Model loading error when running depth/pose evaluation using pretrained models

### ISSUE THREAD

**Issue by jwook1004** [yzcjtr/GeoNet#21]
Hi,

First of all, thanks a lot for sharing your code. I am eager to learn details about your impressive work!

I tried to reproduce the results in the GeoNet paper by running the depth/pose evaluation using the pretrained models. As described in the README, I downloaded "depthnet/dirflownet/posenet/resflownet" from the provided GoogleDrive links. Then I ran the following simple shell script to run evaluation, e.g.,:

```
DATASET_DIR=~/Data/KITTI
INIT_CHECKPOINT_DIR=~/Workspace/GeoNet/models/geonet_depthnet/model.index
OUTPUT_DIR=./Testing_log/depth
python geonet_main.py --mode=test_depth --dataset_dir=$DATASET_DIR --init_ckpt_file=$INIT_CHECKPOINT_DIR --batch_size=1 --depth_test_split=eigen --output_dir=$OUTPUT_DIR
```

Then I got the following error:

> Tensor name "pose_net/Conv/BatchNorm/beta" not found in checkpoint files ~/Workspace/GeoNet/models/geonet_posenet/model.index

I think either I am missing something simple or there's some mismatch in the uploaded python code and the pretrained model. (I got the same error for POSENET evaluation...) I guess this issue could be something any novice like me would suffer!  Could you kindly help me on this?

Best regards,
Jungwook

**Comment by yzcjtr** [comment]
The init checkpoint shouldn't contain any suffix. Use 'model' instead of 'model.index'.
